spec/models/course_spec.rb

```diff
@@ -434,4 +434,103 @@
       its(:content_status) { should eq(:published_with_unpublished_changes) }
     end
   end
+
+  describe "#publish_enrichments" do
+    context 'on a course with only a draft enrichment' do
+      let(:course) do
+        create(:course,
+            changed_at: 10.minutes.ago,
+          with_enrichments: [
+          [:initial_draft, created_at: 1.day.ago, updated_at: 20.minutes.ago],
+        ])
+      end
+      let(:user) { create(:user) }
+
+      it 'should publish the draft' do
```

Comment:
How about nixing the `should`: `it 'publishes the draft'`. For here and below.

Reply:
done, thanks